src/App.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+//import { render } from '@testing-library/react';
+import App from './App.js';
+import Header from './Header.js';
+import ImageItem from './ImageItem.js';
+import ImageList from './ImageList.js';
+
+
+//import Link from '../Link.react';
+import renderer from 'react-test-renderer';
```

Comment:
I don't see `'react-test-renderer'` in your dependencies. Make sure that you're run `npm i react-test-renderer`.